app/views/lessons/project_submissions/index.html.erb

```diff
@@ -1,6 +1,6 @@
 <%= title(@lesson.title) %>
 
-<div class="tmp-container-spacer">
+<div class="page-container">
   <div class="grid grid-cols-12">
     <div class="col-span-12 md:col-start-3 md:col-span-8">
```

Comment:
These column values probably need to be tweaked a little bit now with the new container at different break points (md is probably a bit slim), but it's very minor. Not a blocker

Reply:
Great point, I think we could get away with having this table just be the width of the page container.